repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/mocks/MockIdentityV2Service.groovy

```diff
@@ -781,6 +782,7 @@ class MockIdentityV2Service {
                                                           domain: (values.domain ?: DEFAULT_MAPPING_VALUE),
                                                           name  : (values.name ?: DEFAULT_MAPPING_VALUE),
                                                           email : (values.email ?: DEFAULT_MAPPING_VALUE),
+                                                          groups:  (values.groups ?: DEFAULT_MAPPING_VALUE),
```

Comment:
not critical but there's an extra space

Reply:
Gah. I'll open a separate PR to hit that.